src/tests/data/FactoryUtils.ts

```diff
@@ -25,5 +25,27 @@ class FactoryUtils {
   public static compareByDate(a, b) {
     return new Date(b.date).getTime() - new Date(a.date).getTime();
   }
+
+  /**
+   * Compares the time between two end dates by most recent first
+   *
+   * @param a The first date to be compared
+   * @param b The second date to be compared
+   * @returns 1 if b is more recent than a, -1 if a is more recent than b, 0 if a and b are at the same time
```

Comment:
The `@returns` description here should either state that you return a positive/negative number based on the recentness of the given dates or you should change your function to return +/- 1 (currently you're just returning the difference between the dates in ms).